build-tools/src/main/java/org/elasticsearch/gradle/testclusters/RunTask.java

```diff
@@ -214,7 +214,7 @@ public void beforeStart() {
                 else if (node.getSettingKeys().contains("telemetry.metrics.enabled") == false) { // metrics
                     node.setting("telemetry.metrics.enabled", "false");
                 } else if (node.getSettingKeys().contains("telemetry.tracing.enabled") == false
-                    && node.getSettingKeys().contains("tracing.apm.enabled") == false) { // tracing
+                    && node.getSettingKeys().contains("telemetry.tracing.enable") == false) { // tracing
```

Comment:
Good point, though we have to fix the logic differently to support the fallbacks. You are checking "telemetry.tracing.enable" twice this way

Reply:
fair point. this was stupid find replace.
I don't think we have to worry about deprecated tracing.apm in RunTask
This is a 'dev only' tool.